lib/router.js

```diff
@@ -46,7 +46,7 @@ exports.setupAdminRoutes = function () {
 
 exports.setupModelRoutes = function () {
 
-    router.use('/model/:model', linz.api.middleware.setLinzModel());
+    router.use('/model?/:model', linz.api.middleware.setLinzModel());
```

Comment:
@allanchau, why this change?

Reply:
@smebberson We need to be able to pick up custom routes too, eg `/model/testModel/action/export`. That won't be picked up by `'/model/:model'` in my testing.

Reply:
Do we though? I thought that was the whole point of the changes in `set-linz-model`, it would recognise the `/model/{model}/` pattern?

Reply:
This change means that `model` is optional? I don't see how that would work?

Reply:
Yeah but the middleware still needs to be called? We could keep it as is and require the developer implementing it to call `linz.api.middleware.setLinzModel()` if you prefer?
That would mean all custom export routes need to call that assuming they want to filter based on the list filters though.

Reply:
If the custom export routes are mounted in `linz.app`, this middleware will always run without this `?` customisation....?

Reply:
I think that is the problem though, they aren't mounted in Linz.
Currently developers have to implement it themselves so `req.params.model` is undefined.